src/Features/Core/Portable/ExtractMethod/MethodExtractor.cs

```diff
@@ -173,11 +176,16 @@ public async Task<ExtractMethodResult> ExtractMethodAsync(CancellationToken canc
             return Tuple.Create(true, status);
         }
 
-        internal static string MakeMethodName(string prefix, string originalName)
+        internal static string MakeMethodName(string prefix, string originalName, bool localFunction = false, bool camelCase = false)
```

Comment:
i would not make these optional.  it makes it too easy for existing code to not pass in the right values.